Fix RSF and riot helmet

emmanuelbassil opened the pull request:
🆑 emmanuelbassil
tweak: Rapid service fabricator now uses radial menus
bugfix: Rapid service fabricator properly creates items now.
/🆑

CrimsonShrike commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -69,6 +69,7 @@
 	name = "riot helmet"
 	desc = "It's a helmet specifically designed to protect against close range attacks."
 	icon_state = "helmet_riot"
+	flash_protection = TRUE
```
Use a define (ie, Flash protection minor) as flash has different levels. Also wouldnt sec usually wear flash resistant goggles underneath?

CrimsonShrike replied:
FLASH_PROTECTION_MINOR , for reference, look at other examples

Spookerton commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -69,6 +69,7 @@
 	name = "riot helmet"
 	desc = "It's a helmet specifically designed to protect against close range attacks."
 	icon_state = "helmet_riot"
+	flash_protection = TRUE
```
In fact, just a nope to this one.

Spookerton commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -13,7 +13,7 @@ RSF
 	anchored = FALSE
 	var/stored_matter = 30
-	var/mode = 1
+	var/mode = "Cigarette"
```
swapping these out to strings is an unecessary expense, but I see why because of the radial menu. It's icky, but can be improved by at least using constants (`var/const/MODE_CIGARETTE = "Cigarette"`) instead of hand paired strings.

emmanuelbassil replied:
Fixed

emmanuelbassil committed:
Assorted Fixes

Auto stash before rebase of "refs/heads/Attackby"

Auto stash before rebase of "refs/heads/Stuff"

Auto stash before rebase of "refs/heads/Stuff"
